index.js

```diff
@@ -6,7 +6,7 @@ var markers = [];
 const GLOBAL_MARKER_TYPE = "G";
 const LOCAL_MARKER_TYPE = "L";
 
-L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
+L.tileLayer('https://maps.wikimedia.org/osm-intl/{z}/{x}/{y}.png ', {
     attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
```

Comment:
Please add an attribution to Wikimedia and OSM contributors, like so: https://foundation.wikimedia.org/wiki/Maps_Terms_of_Use#/media/File:Screenshot_of_a_map_on_Wikivoyage_displaying_San_Francisco's_Financial_District.png